storm-core/src/jvm/org/apache/storm/windowing/CountEvictionPolicy.java

```diff
@@ -26,10 +26,10 @@
  * @param <T> the type of event tracked by this policy.
  */
 public class CountEvictionPolicy<T> implements EvictionPolicy<T> {
-    private final int threshold;
+    private final long threshold;
```

Comment:
the `CountWindow` and `DurationWindow` both extend from WindowConfig and `windowLength` and `slideLength` are Long type.